.travis.yml

```diff
@@ -1,4 +1,4 @@
-language: generic
+language: crystal
```

Comment:
This should probably be in a different PR. I know Dru was in favor of using docker which was why I used this fix. Crystal Core team also uses docker for their travis tests.